ACE/ACEXML/apps/svcconf/ACEXML_XML_Svc_Conf_Parser.rc

```diff
@@ -0,0 +1,30 @@
+#include "..\..\..\ace\Version.h"
```

Comment:
Use / instead of \ in case someone tries to cross compile from Linux

Reply:
Updates just pushed.